### ISSUE TITLE
After a round restart, Nuke Op players are spawned in space

### ISSUE THREAD

**Issue by DooblyNoobly** [unitystation/unitystation#1112]
# Preliminary information
After a round finishes and a new one starts, if you choose to spawn as Syndicate role you spawn on the syndicate shuttle for a brief second and then get teleported to some point in space.

## Category
Bug

## Steps to Reproduce
Please enter the steps to reproduce the bug or behaviour:
1. Join the dev build server.
2. Play as syndicate for a whole round
3. Wait for round restart
4. Choose Syndicate role again

**Comment by fomalsd** [comment]
will look into it

**Comment by fomalsd** [comment]
@DooblyNoobly were you able to reproduce it more than once?
Can't seem to reproduce on fake headless server (200 ping)

**Comment by fomalsd** [comment]
Also need more details on what were you doing before round restarted

**Comment by DooblyNoobly** [comment]
Yep can reproduce this everytime and also on different machines. I'm testing on the steam server with the steam builds. 

Last time I tested, I spawned as syndi on the shuttle (outpost dm) I flew the shuttle to the station, got out and ran around. The timer finished (no one killed anyone, pussies). Then the scene reloaded and showed the 'Choose your profession' window. I click Syndicate Role. It spawns on the shuttle for a brief moment and then blinks the player about 100 unity meters away in space where you are stuck and can't do anything. 

Can also reproduce this from Unity Editor. Perhaps this bug doesn't exist on your branch anymore?
